eq-author/src/utils/UrlUtils.js

```diff
@@ -89,6 +90,13 @@ export const buildHistoryPath = ({ questionnaireId }) => {
   });
 };
 
+export const buildPublishPath = ({ questionnaireId }) => {
+  return generatePath(Routes.QUESTIONNAIRE)({
```

Comment:
You could do a single expression return here

Reply:
Would you mind if I don't? Only because it seems like the pattern of similar functions is consistent with that implementation.

Reply:
Spoken to H in person; agreed to keep it as is.